apis/placement/v1beta1/binding_types.go

```diff
@@ -137,6 +137,13 @@ const (
 	// - "False" means not all the resources are created in the target cluster yet.
 	// - "Unknown" means it is unknown.
 	ResourceBindingApplied ResourceBindingConditionType = "Applied"
+
+	// ResourceBindingAvailable indicates the available condition of the given resources.
+	// Its condition status can be one of the following:
+	// - "True" means all the resources are available in the target cluster.
+	// - "False" means not all the resources are available in the target cluster yet.
+	// - "Unknown" means it is unknown.
+	ResourceBindingAvailable ResourceBindingConditionType = "Applied"
```

Comment:
Hi Ryan! I think this should use a different value.

Reply:
And, it's just a nit, but I can't help feeling that the `available` part might have many different interpretations -> though at this moment I couldn't find a more accurate alternative

Reply:
let's discuss this when we start to touch the binding. I will scope this PR to work agent only.